wpilibj/src/main/java/edu/wpi/first/wpilibj/RobotBase.java

```diff
@@ -347,7 +349,9 @@ public static boolean getBooleanProperty(String name, boolean defaultValue) {
       m_runMutex.unlock();
       if (!suppressExitWarning) {
         // startCompetition never returns unless exception occurs....
-        DriverStation.reportWarning("Robots should not quit, but yours did!", false);
+        DriverStation.reportWarning("Robots should not quit, but yours did!"
+          + " Please read the above stacktrace and verify your robot logic."
+          + " Go to https://wpilib.org/stacktrace for help debuging.", false);
```

Comment:
`debugging` is misspelled.

Reply:
`debugging` is spelt correctly

https://en.wikipedia.org/wiki/Debugging
https://www.merriam-webster.com/dictionary/debug

I merged a requested change earlier, so the preview says `debuging` but it will be reverted back